geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -4366,6 +4366,15 @@ public TXManagerImpl getTxManager() {
     return new RegionFactoryImpl<>(this, regionAttributes);
   }
 
+  @Override
+  public <K, V> RegionFactory<K, V> createRegionFactory(RegionFactory<K, V> regionFactory) {
+    throwIfClient();
+    return new RegionFactoryImpl<>(regionFactory);
+  }
+
+  /**
+   * @since GemFire 6.5
```

Comment:
Was this intentional?  Is there a reason why we would put this on the class but not interface?

Reply:
What do you mean? Which API? The above API  for createRegionFactory(RegionFactory<K,V> regionFactory) is in Cache.java

Reply:
I mean, In two parts:

1.) Do we want to annotate this in GemFireCacheImpl (when the other similar methods in GemFireCacheImpl aren't annotated?)  

2.) Shouldn't this be annotated in the Interface instead of the implementing class?

Reply:
I think I have made this discussion moot by removing this functionality and creating a utility class which accesses the now protected RegionFactory method. This means I have removed this API that I added.